### DISCUSSION TITLE
Tab component unexpected behavior

### DISCUSSION THREAD

**Discussion by GonzaCoding** [chakra-ui/chakra-ui#1924] [General] [answered]
Hi!
I tried Tab standard example, but I don't get expected result.
My code:

https://gist.github.com/GonzaCoding/d4df9b3bff67f702cb711ea6ab454282

The result:

![chakra-ui-tab](https://user-images.githubusercontent.com/62549779/91982978-6694cf00-ed01-11ea-9355-23337daf5475.gif)

The border is in the right, not in bottom, also it isn't blue. And it's not white background.

I changed the theme, I suppos it was the problem, but I can't find where to change to get the standard behaivor.
My custom Theme (i have a set of custom icons):

https://gist.github.com/GonzaCoding/24dfc97ef209d369f5d22390cdefb569

Also I tried the example called "Styling the tab states manually", but the behavior is the same.

Thanks!

**Comment by with-heart** [comment, marked as answer]
Hi @GonzaCoding!

Is there any way you could modify this CodeSandbox example to your use case? https://codesandbox.io/s/focused-grass-ph1v8?file=/src/App.tsx

It's hard to tell what the issue could be without a better idea of how you have set things up!

**Reply by GonzaCoding** [reply]
> Hi @GonzaCoding!
> 
> Is there any way you could modify this CodeSandbox example to your use case? https://codesandbox.io/s/focused-grass-ph1v8?file=/src/App.tsx
> 
> It's hard to tell what the issue could be without a better idea of how you have set things up!

hi! I can solve, I forgot CSSReset, and I was no extending base theme, so many styles where forgotten.
Thanks! Can you close this discussion please?
Regards

**Reply by with-heart** [reply]
Glad to hear it's working! There is no closing in discussions, it'll just fade away into history once we stop replying :)
